state/migration_import.go

```diff
@@ -1387,7 +1387,12 @@ func (i *importer) makeCharmOrigin(a description.Application, units []descriptio
 		// Fix the case where `juju set-series` was called before the change to
 		// set the series in the origin's platform as well.
 		// Assumes that UpdateApplicationSeries will not change operating systems.
-		series := p.Series
+		// TODO(wallyworld) - we need to update description to support channel
+		// For now, handle both channel and series here.
+		series, err := coreseries.GetSeriesFromBase(coreseries.Base{Name: p.OS, Channel: p.Channel})
+		if err != nil {
+			series = p.Channel
```

Comment:
maybe log the error?

Reply:
no need because Channel is either a valid version or a series.
will be moot in 3.0 anyway